db/model/collectorgroup.js

```diff
@@ -235,7 +235,7 @@ module.exports = function collectorgroup(seq, dataTypes) {
     return collectorUtils.validate(seq, arr)
       .then(collectorUtils.alreadyAssigned)
       .then((collectors) => this.addCollectors(collectors))
-      .then(() => this.reload())
+      .then(() => this.reload(CollectorGroup.options.defaultScope))
```

Comment:
Less verbose - yes, but it looks more hacky because we are using the variable which looks like it is tagged as private (because it starts with underscore)

Reply:
True. No strong preference, just throwing out an idea. Up to you.